k9mail/src/main/java/com/fsck/k9/provider/MessageProvider.java

```diff
@@ -481,20 +476,6 @@ public Boolean getField(MessageInfoHolder source) {
     }
 
     /**
-     * @deprecated having an incremental value has no real interest, implemented for compatibility only
-     */
```

Comment:
Again, this is an exposed API, not us using deprecated functionality of a library. It's a deprecated API we still support. I'm not convinced of the benefit of removing it.

Reply:
This API was previously unused by any part of the project. It shouldn't have any effect on compatibility, as we don't support binary-compatible releases in any case. I would argue that the fewer unused variables we have the better (by the same reasoning as https://github.com/k9mail/k-9/pull/2753).

Reply:
We support and document an API.

Reply:
I hope by now nobody uses this anymore. But it doesn't hurt to leave it in. I'd rather get rid of `MessageProvider` altogether rather than only parts of it.